package.json

```diff
@@ -47,13 +47,15 @@
     "@adobe/mdast-util-gridtables": "4.0.0",
     "@adobe/remark-gridtables": "3.0.0",
     "cookie": "0.6.0",
+    "dayjs": "1.11.10",
```

Comment:
new dependency (which has no other dependency)

Reply:
realized that we don't need *that* flexibility, but could use a hardcoded date representation (e.g. `YYYY-MM-DD`), so I can get rid of it